diff --git a/dist/index.d.ts b/dist/index.d.ts
index 4a3bf8f6bfa1045d7c76bbcb10773e2fa0bf59b9..828682a74986266aa44e5a3412b9adf2e9d039db 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,6 +1,7 @@
 import * as Vue$1 from 'vue';
 import Vue__default, { VueConstructor, ComponentOptions as ComponentOptions$1, VNode, AsyncComponent } from 'vue';
-import { VueConstructor as VueConstructor$1 } from 'vue/types/umd';
+// import { VueConstructor as VueConstructor$1 } from 'vue/types/umd';
+type VueConstructor$1 = VueConstructor
 
 declare type Data = {
     [key: string]: unknown;